src/main/windows/main.js

```diff
@@ -45,18 +45,26 @@ function init (state, options) {
     y: initialBounds.y
   })
 
-  win.once('ready-to-show', function () {
+  win.loadURL(config.WINDOW_MAIN)
```

Comment:
Does it make a difference where `loadURL` is? IMO it would be safer to have the `ready-to-show` event listener declared first, in case `loadURL` finishes loading very quickly before the listener is added (quite unlikely though).